integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -686,6 +688,10 @@ void testCreateNewCluster() {
    * To: "image: container-registry.oracle.com/middleware/weblogic:DateAndTimeStamp"
    * e.g, From ""image: container-registry.oracle.com/middleware/weblogic:12.2.1.4"
    * To: "image:container-registry.oracle.com/middleware/weblogic:2021-07-08-162571383699"
+   * First time, update the domain with the new image, update restartVersion at the domain level
```

Comment:
Please clarify the purpose 
    Here we want to verify that Rolling Restart can also happen when the Pod are in  "Not Ready" state
Describe Steps as follows ... 
      a. Start a domain with a image does not exists
      b. Update the restart version, the domain should not restart
      c. Make the image available and it should trigger rolling restart

Reply:
Done. Please take another look. Thanks